src/navbar/types.ts

```diff
@@ -60,10 +68,25 @@ export type DrawerDismissalMethod =
 export interface NavbarDrawerProps extends NavbarSharedProps {
     show: boolean;
     children: JSX.Element | JSX.Element[];
-    onClose?: (() => void) | undefined;
+    onClose?: () => void | undefined;
+    resourcesSecondaryBrand?: NavbarResourcesProps | undefined;
```

Comment:
i think `resources` should be refactored to handle the secondary stuff also. I would imagine inside it will have `primary` and `secondary`